### ISSUE TITLE
Include ' in 2' moves.

### ISSUE THREAD

**Issue by drazil100** [spencerchubb/algdb#10]
The direction of a 2' moves is usually intuitive as you would only ever be in a position to rotate in 1 direction, that said it isn't always intuitive and even if that isn't a big issue, having all 2 moves be listed as clockwise rotations messes with the animations. For instance in [OLL 22](https://cubingapp.com/algdb/OLL.html#:~:text=R%27%20U2%20R-,OLL%2022,-R%20U2%20R2) the top algorithm is CLEARLY designed for the first and third R2 to be R2' however the animation only displays clockwise rotations.

I don't expect every algorithm to get fixed but it would be nice if it was at least looked into for the most common algorithms.

**Comment by spencerchubb** [comment]
I added the 'long term' label. I agree this would definitely enhance the database, and the best approach is probably to update things over time as we find them.

**Comment by drazil100** [comment]
Agreed. I imagine we could start with the algs that we already know and use and people can report algs to this thread if there are any that are missed.
